document-store/src/integrationTest/java/org/hypertrace/core/documentstore/postgres/PostgresDocStoreTest.java

```diff
@@ -108,6 +126,41 @@ public void testUpsertAndReturn() throws IOException {
     Assertions.assertEquals(document.toJson(), resultDocument.toJson());
   }
 
+
+  @ParameterizedTest
+  @MethodSource("postgresContextProvider")
+  public void test_getJsonNodeAtPath(String dataStoreName) throws Exception {
+    String inputNode = "{\"id\":\"testKey2\",\"attributes\":{\"name\":\"testKey2\",\"labels\":{\"valueList\":{\"values\":[{\"value\":{\"string\":\"Label2\"}}]}}},\"created_at\":\"2022-07-12 17:46:03.750437\",\"updated_at\":\"2022-07-12 17:46:03.750437\"}";
```

Comment:
Can you create a JSON using an object mapper instead? Look at how it's being done right now. This is to improve readability.

Reply:
see the next line..i have used objectMapper to create json node